pkgs/development/python-modules/gurobipy/linux.nix

```diff
@@ -1,19 +1,19 @@
-{ fetchurl, python }:
+{ lib, fetchurl, python }:
 assert with python.pkgs; isPy27 || isPy36 || isPy37 || isPy38 || isPy39;
```

Comment:
For what do we need this line?

Reply:
I thought it was good practice to add since `setup.py` defines those as the compatible versions, but now I realize that `setup.py` when run performs that check already, so I will remove it.